pkgs/development/tools/database/prisma-engines/default.nix

```diff
@@ -28,7 +28,7 @@ rustPlatform.buildRustPackage rec {
     lockFile = ./Cargo.lock;
```

Comment:
Could you change this to use `fetchCargoVendor` that was introduced in https://github.com/NixOS/nixpkgs/pull/349360